pkg/codegen/model.go

```diff
@@ -1,6 +1,7 @@
 package codegen
 
 import (
+	"github.com/atombender/go-jsonschema/pkg/schemas"
```

Comment:
Don't know about the formatting requirements here but this should probably go after the standard library imports.

Reply:
is there a way to tell Goland about this? govet also doesn't pick that up somehow

Reply:
Settings->Editor->Code Style->Go and there you can specify import order and switch over from `gofmt` to `goimports`.